user-guide/Basic_Functionality/Visio/tutorials/Using_Visual_Overview_to_switch_spectrum_presets.md

```diff
@@ -0,0 +1,195 @@
+---
+uid: Using_Visual_Overview_to_switch_spectrum_presets
+---
+
+# Using Visual Overview to switch spectrum presets
+
+This tutorial explains how to start from a blank Visio file and add a spectrum control and several buttons to switch between presets.
+
+Expected duration: 25 minutes
+
+## Prerequisites
+
+- A DataMiner System that is [connected to dataminer.services](xref:Connecting_your_DataMiner_System_to_the_cloud) and where the connector  [Skyline Spectrum Simulator](https://catalog.dataminer.services/details/6f33ec9f-e83d-49d5-8f85-87ad66eaa5c7) has been deployed.
+- [Microsoft Visio](https://www.microsoft.com/microsoft-365/visio/flowchart-software)
```

Comment:
It might be better if we also mention which version of Microsoft Visio, so people using ancient versions don't try to do the tutorial and then run into trouble.

Reply:
Used version added.